<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="UnhandledExceptionEventArgs.xml" source-language="en-US" target-language="it-IT">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac52bc5b27ab8c70f87bc485bcd049f405e98420956.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">2bc5b27ab8c70f87bc485bcd049f405e98420956</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.UnhandledExceptionEventArgs">
          <source>Provides data for the event that is raised when there is an exception that is not handled in any application domain.</source>
          <target state="translated">Fornisce dati relativi all'evento generato quando si verifica un'eccezione non gestita in qualsiasi dominio di applicazione.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.UnhandledExceptionEventArgs">
          <source><ph id="ph1">&lt;xref:System.UnhandledExceptionEventArgs&gt;</ph> provides access to the exception object and a flag indicating whether the common language runtime is terminating.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.UnhandledExceptionEventArgs&gt;</ph> fornisce l'accesso all'oggetto eccezione e un flag che indica se common language runtime sta terminando.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.UnhandledExceptionEventArgs">
          <source>The <ph id="ph1">&lt;xref:System.UnhandledExceptionEventArgs&gt;</ph> is one of the parameters passed into <ph id="ph2">&lt;xref:System.UnhandledExceptionEventHandler&gt;</ph> for the <ph id="ph3">&lt;xref:System.AppDomain.UnhandledException?displayProperty=nameWithType&gt;</ph> event.</source>
          <target state="translated">Il <ph id="ph1">&lt;xref:System.UnhandledExceptionEventArgs&gt;</ph> è uno dei parametri passati a <ph id="ph2">&lt;xref:System.UnhandledExceptionEventHandler&gt;</ph> per il <ph id="ph3">&lt;xref:System.AppDomain.UnhandledException?displayProperty=nameWithType&gt;</ph> evento.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.UnhandledExceptionEventArgs">
          <source>For more information about event handler delegates, see <bpt id="p1">[</bpt>Handling and Raising Events<ept id="p1">](~/docs/standard/events/index.md)</ept>.</source>
          <target state="translated">Per ulteriori informazioni sui delegati del gestore eventi, vedere <bpt id="p1">[</bpt>la gestione e generazione di eventi<ept id="p1">](~/docs/standard/events/index.md)</ept>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.UnhandledExceptionEventArgs.#ctor(System.Object,System.Boolean)">
          <source>The exception that is not handled.</source>
          <target state="translated">Eccezione non gestita.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.UnhandledExceptionEventArgs.#ctor(System.Object,System.Boolean)">
          <source><ph id="ph1">&lt;see langword="true" /&gt;</ph> if the runtime is terminating; otherwise, <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;see langword="true" /&gt;</ph> se la fase di esecuzione sta terminando; in caso contrario <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.UnhandledExceptionEventArgs.#ctor(System.Object,System.Boolean)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.UnhandledExceptionEventArgs" /&gt;</ph> class with the exception object and a common language runtime termination flag.</source>
          <target state="translated">Consente di inizializzare una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.UnhandledExceptionEventArgs" /&gt;</ph> con l'oggetto eccezione e un flag di terminazione di Common Language Runtime.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>Gets the unhandled exception object.</source>
          <target state="translated">Ottiene l'oggetto eccezione non gestita.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>The unhandled exception object.</source>
          <target state="translated">Oggetto eccezione non gestita.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>This property returns an object of type <ph id="ph1">&lt;xref:System.Object&gt;</ph> rather than one derived from <ph id="ph2">&lt;xref:System.Exception&gt;</ph>.</source>
          <target state="translated">Questa proprietà restituisce un oggetto di tipo <ph id="ph1">&lt;xref:System.Object&gt;</ph> anziché uno derivato da <ph id="ph2">&lt;xref:System.Exception&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>Although the Common Language Specification requires that all exception types derive from <ph id="ph1">&lt;xref:System.Exception&gt;</ph>, it is possible for methods to throw exceptions with objects not derived from <ph id="ph2">&lt;xref:System.Exception&gt;</ph>.</source>
          <target state="translated">Anche se è necessario che tutti i tipi di eccezione deve derivare da Common Language Specification <ph id="ph1">&lt;xref:System.Exception&gt;</ph>, è possibile chiamare i metodi generare eccezioni con gli oggetti non derivati da <ph id="ph2">&lt;xref:System.Exception&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>You can do the following to work with this exception:</source>
          <target state="translated">È possibile eseguire il comando seguente per lavorare con l'eccezione seguente:</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>Apply the  <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute&gt;</ph> attribute with a <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute.WrapNonExceptionThrows%2A?displayProperty=nameWithType&gt;</ph> value of <ph id="ph3">`true`</ph> to the assembly that contains the event handler.</source>
          <target state="translated">Applicare il <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute&gt;</ph> attributo con un <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute.WrapNonExceptionThrows%2A?displayProperty=nameWithType&gt;</ph> valore <ph id="ph3">`true`</ph> all'assembly che contiene il gestore dell'evento.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>This wraps all exceptions not derived from the <ph id="ph1">&lt;xref:System.Exception&gt;</ph> class in a <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException&gt;</ph> object.</source>
          <target state="translated">Si esegue il wrapping di tutte le eccezioni non derivate dal <ph id="ph1">&lt;xref:System.Exception&gt;</ph> classe un <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException&gt;</ph> oggetto.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>You can then safely cast (in C#) or convert (in Visual Basic) the object returned by this property to an <ph id="ph1">&lt;xref:System.Exception&gt;</ph> object, and retrieve the original exception object from the <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException%2A?displayProperty=nameWithType&gt;</ph> property.</source>
          <target state="translated">È possibile quindi in modo sicuro il cast (in c#) o convertire (in Visual Basic) l'oggetto restituito da questa proprietà per un <ph id="ph1">&lt;xref:System.Exception&gt;</ph> dell'oggetto e recuperare l'oggetto eccezione originale il <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException%2A?displayProperty=nameWithType&gt;</ph> proprietà.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>Note that some compilers, such as the C# and Visual Basic compilers, automatically apply this attribute.</source>
          <target state="translated">Si noti che alcuni compilatori, ad esempio i compilatori c# e Visual Basic, applicano automaticamente questo attributo.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>Cast the object returned by this property to an <ph id="ph1">&lt;xref:System.Exception&gt;</ph> object.</source>
          <target state="translated">Il cast dell'oggetto restituito da questa proprietà per un <ph id="ph1">&lt;xref:System.Exception&gt;</ph> oggetto.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>The following example demonstrates the <ph id="ph1">&lt;xref:System.AppDomain.UnhandledException&gt;</ph> event.</source>
          <target state="translated">Nell'esempio seguente viene illustrato il <ph id="ph1">&lt;xref:System.AppDomain.UnhandledException&gt;</ph> evento.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>It defines an event handler, <ph id="ph1">`MyHandler`</ph>, that is invoked whenever an unhandled exception is thrown in the default application domain.</source>
          <target state="translated">Definisce un gestore eventi, <ph id="ph1">`MyHandler`</ph>, che viene richiamato ogni volta che viene generata un'eccezione non gestita nel dominio applicazione predefinito.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>It then throws two exceptions.</source>
          <target state="translated">Genera quindi due eccezioni.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>The first is handled by a <bpt id="p1">**</bpt>try/catch<ept id="p1">**</ept> block.</source>
          <target state="translated">Il primo viene gestito da un <bpt id="p1">**</bpt>try/catch<ept id="p1">**</ept> blocco.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.ExceptionObject">
          <source>The second is unhandled and invokes the <ph id="ph1">`MyHandle`</ph> routine before the application terminates.</source>
          <target state="translated">Il secondo viene gestito e richiama il <ph id="ph1">`MyHandle`</ph> routine prima della chiusura dell'applicazione.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source>Indicates whether the common language runtime is terminating.</source>
          <target state="translated">Indica se Common Language Runtime sta terminando.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source><ph id="ph1">&lt;see langword="true" /&gt;</ph> if the runtime is terminating; otherwise, <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;see langword="true" /&gt;</ph> se la fase di esecuzione sta terminando; in caso contrario <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source>Beginning with the .NET Framework version 2.0, this property returns <ph id="ph1">`true`</ph> for most unhandled exceptions, unless an application compatibility flag is used to revert to the behavior of versions 1.0 and 1.1.</source>
          <target state="translated">A partire da .NET Framework versione 2.0, questa proprietà restituisce <ph id="ph1">`true`</ph> per la maggior parte le eccezioni non gestite, a meno che non viene utilizzato un flag di compatibilità dell'applicazione per ripristinare il comportamento delle versioni 1.0 e 1.1.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source>The reason is that most unhandled exceptions in threads, including thread pool threads, are allowed to proceed naturally, which normally results in termination of the application.</source>
          <target state="translated">Il motivo è che è consentite proseguire normalmente, più eccezioni non gestite nei thread, inclusi i thread del pool, che in genere comporta la terminazione dell'applicazione.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source>For more information, including a list of cases in which the runtime does not terminate, see <bpt id="p1">[</bpt>Exceptions in Managed Threads<ept id="p1">](~/docs/standard/threading/exceptions-in-managed-threads.md)</ept>.</source>
          <target state="translated">Per ulteriori informazioni, incluso un elenco di casi in cui il runtime non viene terminato, vedere <bpt id="p1">[</bpt>eccezioni in thread gestiti<ept id="p1">](~/docs/standard/threading/exceptions-in-managed-threads.md)</ept>.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source>Version Information</source>
          <target state="translated">Informazioni sulla versione</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source>The following applies to the .NET Framework versions 1.0 and 1.1: This property returns <ph id="ph1">`true`</ph> when an exception is received in, but not handled by, an application's main thread.</source>
          <target state="translated">Quanto segue si applica alle versioni di .NET Framework 1.0 e 1.1: questa proprietà restituisce <ph id="ph1">`true`</ph> quando viene ricevuta in, un'eccezione ma non è gestita da, thread principale di un'applicazione.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source>This property also returns <ph id="ph1">`true`</ph> if an unmanaged thread is executing managed code and receives an unhandled exception.</source>
          <target state="translated">Questa proprietà restituisce inoltre <ph id="ph1">`true`</ph> se un thread non gestito esegue codice gestito e riceve un'eccezione non gestita.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.UnhandledExceptionEventArgs.IsTerminating">
          <source>Finally, this property returns <ph id="ph1">`false`</ph> for thread pool threads and managed threads created by an application, because versions 1.0 and 1.1 catch unhandled exceptions in such threads and ignore them.</source>
          <target state="translated">Infine, questa proprietà restituisce <ph id="ph1">`false`</ph> per i thread del pool e i thread gestiti creati da un'applicazione, perché le versioni 1.0 e 1.1 rilevano le eccezioni non gestite in tali thread e ignorano.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>